### ISSUE TITLE
[Google Cloud Platform Search] Can't search project

### ISSUE THREAD

**Issue by PixiBixi** [raycast/extensions#14342]
### Extension

https://www.raycast.com/klaa97/gcp-search

### Raycast Version

1.82.2

### macOS Version

14.6.1

#### Error:
```
RangeError: "length" is outside of buffer bounds

Buffer.utf8Write:buffer:1066:13
Sl.SD:searchProjects.js:5:28309
we.finish:searchProjects.js:5:27525
searchProjects.js:15:42847
```

### Steps To Reproduce

1. Search "Search Project" on Raycast
2. Wait for the bug

**Comment by klaa97** [comment]
Solved with #14341 

@raycastbot close this issue

**Comment by klaa97** [comment]
@raycastbot close this issue
